src/lib/segments/segment-service-interface.ts

```diff
@@ -13,7 +13,18 @@ export interface ISegmentService {
 
     get(id: number): Promise<ISegment>;
 
-    getStrategies(id: number, userId: number): Promise<IFeatureStrategy[]>;
+    /**
+     * Gets all strategies for a segment
+     * This is NOT considering the private projects
+     * For most use cases, use `getVisibleStrategies`
+     * @param id segment id
```

Comment:
do we need this @param is we have typescript?

Reply:
Sure, no.